.github/workflows/ci.yml

```diff
@@ -111,3 +111,49 @@ jobs:
           name: '${{ env.file_name }}'
           path: '/home/runner/work/guibot/guibot/${{ env.file_name }}'
         if: matrix.install_variant != 'pip'
+
+  pip-wheels:
+    name: Pip wheels with Python ${{ matrix.python-version }} on ${{ matrix.install_variant }}
+    runs-on: ubuntu-latest
+    env:
+      DISPLAY: ':99.0'
+
+    strategy:
+      matrix:
+        python-version: [ 3.12 ]
+        install_variant: [ "pip.fedora.40" ]
+      fail-fast: false
+
+    steps:
+      - run: echo "Job triggered by a ${{ github.event_name }} event on branch is ${{ github.ref }} in repository is ${{ github.repository }}, runner on ${{ runner.os }}"
+      - name: Check out repository code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run pip packager and test coverage
+        run: |
+          cd packaging 
+          bash packager_docker.sh
+        env:
+          INSTALL_VARIANT: ${{ matrix.install_variant }}
+          TESSDATA_PREFIX: /usr/share/tesseract-ocr/5/tessdata
+          PYTHON_VERSION: ${{ matrix.python-version }}
+
+      - name: Set Artifacts and Release Variables
+        id: set_vars
+        run: |
+          VERSION="$( cat /home/runner/work/guibot/guibot/VERSION )"
+          echo "version=$VERSION" >> $GITHUB_ENV
+          WHEEL_NAME="guibot-${VERSION}.1-py3-none-any.whl"
+          echo "file_name=$WHEEL_NAME" >> $GITHUB_ENV
+        if: matrix.install_variant != 'pip'
+
+      - name: Upload Artifacts Wheels
```

Comment:
"Upload artifact wheels" - no need of capitalization of each word here and each step above.